core/CMakeLists.txt

```diff
@@ -2,16 +2,47 @@ cmake_minimum_required(VERSION 3.10)
 
 set(CODSPEED_VERSION 1.2.0)
 
-project(codspeed VERSION ${CODSPEED_VERSION} LANGUAGES CXX)
+project(codspeed VERSION ${CODSPEED_VERSION} LANGUAGES CXX C)
 
 # Specify the C++ standard
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 # Add the include directory
 include_directories(include)
+include_directories(instrument-hooks/includes)
 
 # Add the library
+add_library(
+    instrument_hooks
+    instrument-hooks/dist/core.c
+)
+
+# Suppress warnings for the instrument_hooks library
+if(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
```

Comment:
kind of the same remark here, shouldn't this be exported as part of instrument hook's dist ?